src/aosm/azext_aosm/cli_handlers/onboarding_core_vnf_handler.py

```diff
@@ -146,6 +146,9 @@ def _get_default_config(self, vhd) -> Dict[str, Any]:
             default_config.update({"image_hyper_v_generation": "V1"})
         if vhd.image_api_version:
             default_config.update({"image_api_version": vhd.image_api_version})
+            
+        # Add imageName
+        default_config["imageName"] = self.config.nf_name + 'Image'
```

Comment:
Why do we have this and (basically) the same code on line 75?

Reply:
One is for the vhd and one is for the arm template